rmc-docs/src/install-guide.md

```diff
@@ -0,0 +1,85 @@
+# RMC Installation Guide
+
+RMC has not yet reached the point where releases are available, and so to use RMC you must currently build from source.
+
+In general, the following dependencies are required:
+
+1. The dependencies needed to built `rustc`. RMC is a fork of the Rust compiler, and so we have the same minimum requirements.
```

Comment:
Maybe add that `cargo` will handle most of the installation for these dependencies?